pkg/volume/azure_dd/azure_mounter.go

```diff
@@ -102,7 +102,6 @@ func (m *azureDiskMounter) SetUpAt(dir string, mounterArgs volume.MounterArgs) e
 			klog.Errorf("azureDisk - Unmount directory %s failed with %v", dir, err)
 			return err
 		}
-		mountPoint = true
```

Comment:
Can you share more about how we can be confident this assignment is safe to delete?

Reply:
@mattjmcnaughton sure. mountPoint's value previously assigned on line 105 is not being read anywhere after this point. It gets reassigned anew on this line https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/azure_dd/azure_mounter.go#L139

Reply:
Hmm - I'm far from an expert on this part of the code base... but is `mountPoint = true` an truthful reflection of the state of the world at line 105 and after? If yes, it still feels useful to have be accurate, even if its not being used now.

Reply:
@mattjmcnaughton I can put it back if it's a deal breaker